lib/__testfixtures__/window.output.js

```diff
@@ -1,7 +1,9 @@
-import { setTimeout as Window_setTimeout, parseFloat as Window_parseFloat, closed_get as Window_closed_get, status_set as Window_status_set } from "std:global/Window";
+import { setTimeout as Window_setTimeout, parseFloat as Window_parseFloat, closed_get as Window_closed_get, status_set as Window_status_set, status_get as Window_status_get } from "std:global/Window";
 import { apply as Reflect_apply } from "std:global/Reflect";
 Window_setTimeout(() => {
 }, 3000);
 const f = Window_parseFloat('5.5');
 const a = Reflect_apply(Window_closed_get, window);
 Reflect_apply(Window_status_set, window, ['open']);
+Reflect_apply(Window_status_set, window, ['closed']);
+const s = Reflect_apply(Window_status_get, window);
```

Comment:
OK, so seeing these results made me realize something. `window` is itself a getter, so we don't want it to appear in the output. Instead we want to do `Window_closed_get()` and `Window_status_set('closed')`. This is a very special case. Note that this only works because of https://heycam.github.io/webidl/#dfn-attribute-getter 1.1.2.1 or https://heycam.github.io/webidl/#dfn-attribute-setter 1.4.5.1.

We can either do that as part of this PR, or as a separate one on top. If a separate one, then let's create a new issue to track.

Note that this should apply to both the "bare" case and the non-bare case.

Reply:
I set up an #49 to track this, but I think it can relatively simply be added to this PR

Reply:
Going to fix this in another PR, I just looked into it and it's pretty simple to add for the bare case, but more complicated for the non-bare case. Worth splitting into its own CL

Reply:
Never mind 😅, adding this turned out to be relatively simple, and simplified some of the rest of the logic as well